include/index_config.h

```diff
@@ -0,0 +1,196 @@
+#include "common_includes.h"
+#include "parameters.h"
+
+namespace diskann
+{
+enum LoadStoreStrategy
```

Comment:
Separate enums for DataStoreStrategy and GraphStoreStrategy.
enum should enumerate only supported strategies.